### ISSUE TITLE
In Identity UI, "Logout" button does not show focus state

### ISSUE THREAD

**Issue by SteveSandersonMS** [dotnet/aspnetcore#25675]
This was reported as an accessibility issue at https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1162890

In `Logout.cshtml`, the logout button is defined as:

```html
<button type="submit" class="nav-link btn btn-link text-dark">Click here to Logout</button>
```

The combination of `nav-link` with `btn-link` unfortunately has the bad behavior that it doesn't display any focus indication when focused using the keyboard. `nav-link` tries to set a text-underline style when focused, but `btn-link` overrides that and sets text-decoration to none in all cases.

Some other combination of styles should be used. Or it would be possible to fix it by defining some custom CSS that draws a focus highlight around this element when focused, but that would be less desirable because where possible we want the styles to come from Bootstrap, not custom overrides.

**Comment by SteveSandersonMS** [comment]
In case we need to fix this for 5.0, I suggest adding the following custom CSS:

```css
.btn-link.nav-link:focus {
  outline: black auto 1px;
}
```

**Comment by blowdart** [comment]
@HaoK We added this recently didn't way? Can we look at a different combination of bootstrap styles, rather than custom CSS?

**Comment by surayya-MS** [comment]
Already fixed in this PR https://github.com/dotnet/aspnetcore/pull/25678
